configuration/config-txt/boot.md

```diff
@@ -99,12 +99,37 @@ If `disable_splash` is set to `1`, the rainbow splash screen will not be shown o
 
 On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be routed to the ARM cores using the legacy interrupt controller, rather than via the GIC-400. The default value is `1`.
 
+## os_prefix
+<a name="os_prefix">
+
+The value in `os_prefix` is prepended to (stuck in front of) the name of any operating system files loaded by the firmware, where "operating system files" is defined to mean kernels, initramfs, cmdline.txt, .dtbs and overlays. The prefix would commonly be a directory name, but it could also be part of the filename such as "test-". For this reason, directory prefixes must include the trailing `/` character.
```

Comment:
If I've understood correctly, it sounds like `os_prefix` defaults to "" - is it worth explicitly mentioning that?

Reply:
Yes, effectively it does, but I'd rather address this with a short preamble explaining that use of os_prefix is optional.